Tuist/ProjectDescriptionHelpers/AppCustomConfiguration.swift

```diff
@@ -0,0 +1,107 @@
+import ProjectDescription
+
+public enum AppCustomConfiguration {
+    case debug, betaDevelopment, betaStage, betaProduction, release
+    
+    internal func customConfiguration(with name: String, projectVersion: Version) -> CustomConfiguration {
+        switch self {
+        case .debug:
+            return CustomConfiguration.debug(name: configurationName, settings: settings(with: name, projectVersion: projectVersion))
+        case .betaDevelopment, .betaProduction, .betaStage, .release:
+            return CustomConfiguration.release(name: configurationName, settings: settings(with: name, projectVersion: projectVersion))
+        }
+    }
+    
+    internal func customTargetConfiguration(with name: String) -> CustomConfiguration {
+        switch self {
+        case .debug:
+            return CustomConfiguration.debug(name: configurationName, settings: targetSettings(with: name))
+        case .betaDevelopment, .betaProduction, .betaStage, .release:
+            return CustomConfiguration.release(name: configurationName, settings: targetSettings(with: name))
+        }
+    }
+    
+    private var configurationName: String {
+        switch self {
+        case .debug:
+            return "Debug"
+        case .betaDevelopment:
+            return "Beta-Development"
+        case .betaStage:
+            return "Beta-Stage"
+        case .betaProduction:
+            return "Beta-Production"
+        case .release:
+            return "Release"
+        }
+    }
+    
+    private func settings(with name: String, projectVersion: Version) -> [String: SettingValue] {
+        let base: [String: SettingValue] = [
+            "ACK_ENVIRONMENT_DIR": "$(PROJECT_DIR)/$(TARGET_NAME)/Environment",
+            "ACK_PROJECT_VERSION": SettingValue(stringLiteral: projectVersion.description),
+            "ACK_PRODUCT_NAME": SettingValue(stringLiteral: name),
+            "DEVELOPMENT_TEAM": "PXDF48X6VX",
+            "OTHER_LDFLAGS": "-ObjC",
+            "ENABLE_BITCODE": "NO",
+            "INFOPLIST_PREPROCESS": "YES",
+            "INFOPLIST_PREFIX_HEADER": "$(ACK_ENVIRONMENT_DIR)/.environment_preprocess.h",
+            "CODE_SIGN_STYLE": "Manual",
+        ]
+        
+        switch self {
+        case .debug:
+            let debugSettings: [String: SettingValue] = [
+                "ACK_BUNDLE_ID": SettingValue(stringLiteral: "cz.ackee.enterprise.\(name).\(identifierName)"),
```

Comment:
Do we still need `ACK_BUNDLE_ID`? It's same as `PRODUCT_BUNDLE_IDENTIFIER` and it has no additional value. Or am I wrong? 🤔

Reply:
Yep, you're right, changed